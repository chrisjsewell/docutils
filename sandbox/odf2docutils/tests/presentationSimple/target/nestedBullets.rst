Slide title
===========

* First bullet

  * Nested L1

  * Again L1

    * Nested L2

* Second bullet

  * Second L1

    * Now L2

    * Again L2

      * L3
