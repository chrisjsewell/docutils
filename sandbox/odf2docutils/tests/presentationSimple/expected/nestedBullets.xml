<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE document PUBLIC "+//IDN docutils.sourceforge.net//DTD Docutils Generic//EN//XML" "http://docutils.sourceforge.net/docs/ref/docutils.dtd">
<document source="input/nestedBullets.odp"><section><title>Slide title</title><bullet_list bullet="*"><list_item><paragraph>First bullet</paragraph><bullet_list bullet="*"><list_item><paragraph>Nested L1</paragraph></list_item><list_item><paragraph>Again L1</paragraph><bullet_list bullet="*"><list_item><paragraph>Nested L2</paragraph></list_item></bullet_list></list_item></bullet_list></list_item><list_item><paragraph>Second bullet</paragraph><bullet_list bullet="*"><list_item><paragraph>Second L1</paragraph><bullet_list bullet="*"><list_item><paragraph>Now L2</paragraph></list_item><list_item><paragraph>Again L2</paragraph><bullet_list bullet="*"><list_item><paragraph>L3</paragraph></list_item></bullet_list></list_item></bullet_list></list_item></bullet_list></list_item></bullet_list></section></document>
